docs/configuration.md

```diff
@@ -694,7 +694,7 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
   <td><code>spark.executor.logs.rolling.strategy</code></td>
-  <td>(none)</td>
+  <td>""</td>
```

Comment:
Then it seems like `""`. It is disabled by default so I think it's not super confusing?

Reply:
@HyukjinKwon Thank you so much for your review.
I think it is unusual. it should be different from other situations, jush like which uses Nil on it's default value. This is just my opinion. Just like you said, it's not super confusing. 
Thank you again. Please decide whether to support this pr.

Reply:
You could write `"" (disabled)` to try to both express the default value and reaffirm what it means. But it already says the default is disabled and (none) communicates "not set to anything". I would leave it

Reply:
@srowen Thank you for your advice.
I had submit a new commit to express the default value and reaffirm the meaning.